rpm/kernel-adaptation-pc.spec

```diff
@@ -92,13 +55,9 @@ find %{buildroot}/lib/modules/%{kernel_version_build} -name "*.ko" -type f -exec
 mkdir -p %{buildroot}/boot/
 make INSTALL_PATH=%{buildroot}/boot/ install
 
-%if 0%{?builds_uImage}
-install -m 755 arch/%{kernel_arch}/boot/uImage %{buildroot}/boot/
-%endif
-
-%if 0%{?builds_vmlinuz}
 install -m 755 arch/%{kernel_arch}/boot/bzImage %{buildroot}/boot/vmlinuz-%{kernel_version_build}
-%endif
+# We dont need the vmlinuz package itself.
+rm -f %{buildroot}/boot/vmlinuz
```

Comment:
I think we should instead omit the make install invocation. See kernel/arch/x86/boot/install.sh - seems useless.